internal/agent/agent.go

```diff
@@ -101,9 +102,26 @@ func (a *Agent) Run(ctx context.Context) error {
 
 	executer := &executer.CommonExecuter{}
 
+	// TODO: expose through config
```

Comment:
I would prefer that this go through the fleet/device config (not relevant for this PR of course)